.gitignore

```diff
@@ -26,3 +26,9 @@ vignettes/*.pdf
 *.knit.md
 # Shiny token, see https://shiny.rstudio.com/articles/shinyapps.html
 rsconnect/
+/doc/
+/Meta/
+
+.Rbuildignore
+
+*.Rproj
```

Comment:
Maybe add `*.bib.sav` and remove `references.bib.sav` as I don't think these are needed in the repo.

Reply:
Resolved.